test/metadataSpec.js

```diff
@@ -26,6 +26,7 @@ describe('Metadata', function () {
           expect(res.body).to.deep.equal({
             currency_code: null,
             currency_symbol: null,
+            ilp_prefix: null,
```

Comment:
Yes but it doesn't test to make sure that when you set it it actually shows up in the metadata. Testing if a value is null in this case doesn't actually test anything